### PR TITLE
fix(DB/Item): Epic TBC gems require jewelcrafting

### PR DESCRIPTION
**Pull request by Maria-sequel**
## Changes Proposed:
This PR proposes changes to:
-  [ ] Core (units, players, creatures, game systems).
-  [ ] Scripts (bosses, spell scripts, creature scripts).
-  [x] Database (SAI, creatures, etc).

If your pull request promotes complex changes that require a detailed explanation, please describe them in detail specifying what your solution is and what is it meant to address.

## Issues Addressed:

- Closes https://github.com/chromiecraft/chromiecraft/issues/6050

## SOURCE:

The changes have been validated through:
- [ ] Live research (checked on live servers, e.g Classic WotLK, Retail, etc.)
- [ ] Sniffs (remember to share them with the open source community!)
- [x] Video evidence, knowledge databases or other public sources (e.g forums, Wowhead, etc.)
- [ ] The changes promoted by this pull request come partially or entirely from another project (cherry-pick). **Cherry-picks must be committed using the proper --author tag in order to be accepted, thus crediting the original authors, unless otherwise unable to be found**

## Tests Performed:

This PR has been:
- [ ] Tested in-game by the author.
- [ ] Tested in-game by other community members/someone else other than the author/has been live on production servers.
- [ ] This pull request requires further testing and may have edge cases to be tested.

## How to Test the Changes:

- [ ] This pull request can be tested by following the reproduction steps provided in the linked issue
- [ ] This pull request requires further testing. Provide steps to test your changes. If it requires any specific setup e.g multiple players please specify it as well.

1.
2.
3.

## Known Issues and TODO List:

- [ ]
- [ ]

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/needjc.sql [hunk 1]
```diff
@@ -0,0 +1,2 @@
+--
+UPDATE `item_template` SET `RequiredSkill` = 755, `RequiredSkillRank` = 360 WHERE `entry` IN (33131, 33133, 33134, 33135, 33140, 33143, 33144);
```
Comment (Gultask) [inline data/sql/updates/pending_db_world/needjc.sql:2]: Should be sniffed: https://www.azerothcore.org/wiki/sniffing-and-parsing
Changed in WotLK, maybe

Comment (Gultask) [reply data/sql/updates/pending_db_world/needjc.sql:2]: Although it looks like tbcMangos has it 0'ed as well

Comment (Maria-sequel) [reply data/sql/updates/pending_db_world/needjc.sql:2]: https://www.wowhead.com/wotlk/item=33131/crimson-sun

Comment (Gultask) [reply data/sql/updates/pending_db_world/needjc.sql:2]: https://web.archive.org/web/20110128042900/http://www.wowhead.com:80/item=33131

https://wowwiki-archive.fandom.com/wiki/Crimson_Sun?oldid=1816786

Comment (Maria-sequel) [reply data/sql/updates/pending_db_world/needjc.sql:2]: image
proofs your source is invalid. missing requires master hammersmith

Comment (Gultask) [reply data/sql/updates/pending_db_world/needjc.sql:2]: image

https://www.wowhead.com/wotlk/item=33131/crimson-sun#comments:id=225584

:)

@Maria-sequel could you make a PR in mod-prog instead?

Comment (Maria-sequel) [reply data/sql/updates/pending_db_world/needjc.sql:2]: Comments on 3.2 say it wasn't the case

Comment (Maria-sequel) [reply data/sql/updates/pending_db_world/needjc.sql:2]: Anyway no point in this. I am right and you want sniffs so go make it yourself

**Pull request closed without merging**